intermediate-javascript-calorie-counter/index170.js

```diff
@@ -0,0 +1,21 @@
+document.getElementById('calorie-form').onsubmit = calculate;
+
+function calculate(e) {
+  e.preventDefault();
+
+  const total = Array.from(document.getElementsByClassName('cal-control'))
+    .map(meal => Number(meal.value))
+    .reduce((accumulator, currentValue) => {
+      /*code to run*/
+    });
+}
+/*
+The reduce method takes a callback function as well as an optional initial value, its function signature is as follows:
```

Comment:
Provide the number zero as the initial value of the `reduce()` function.
Here is an example of a reduce function with an empty object as its initial value: 
```
arr.reduce((accumulator, currentValue) => {
  /*code to run*/
}, {});
```